### DISCUSSION TITLE
Visual light communication

### DISCUSSION THREAD

**Discussion by zubow** [NVlabs/sionna#395] [General]
Is it possible to adapt Sionna to simulate free-space optical, e.g. infrared light (300 GHz – 430 THz)?

**Comment by jhoydis** [comment]
Hi @zubow,

If you want to do ray tracing in this frequency range, you would need to be able to characterize material properties.
If you only care about free-space without any obstacles, I see not issue.

Hope that helps.

**Comment by zubow** [comment]
Thank you for for quick answer. Just to be sure: if I want to simulate reflections from obstacles in the THz range, then I just have to adjust the material constants (https://nvlabs.github.io/sionna/api/rt.html#radio-materials)? How do I simulate field of view at TX and RX side? Adjusting the antenna pattern?

**Reply by jhoydis** [reply]
Yes, for the materials. I am not sure what the field of view is in optical communications. Most likely, it is similar to an antenna pattern. The other thing to keep in mind is that you will probably not exploit polarization as well as phase information at all.
